www/js/viewer.js

```diff
@@ -298,21 +298,50 @@ const createCards = (rows, LineID) => {
     gurmukhiContainer.innerHTML = `<span class="padchhed">${padched}</span>
                                     <span class="larivaar">${larivaar}</span>`;
 
-    const englishContainer = document.createElement('div');
-    englishContainer.innerHTML = row.English || '';
+    const enTranslation = h('div.english-translation.transtext');
+    enTranslation.innerHTML = row.English || '';
+    const esTranslation = h('div.spanish-translation.transtext');
+    // If spanish translation is not available fall back to english
+    esTranslation.innerHTML = row.Spanish || row.English;
+
+    const translationsContainer = document.createElement('div');
+    translationsContainer.appendChild(enTranslation);
+    translationsContainer.appendChild(esTranslation);
+
+    const shTransliteration = h(
+      'div.shahmukhi-transliteration.translittext',
+      row.Transliteration.Shahmukhi || '',
+    );
+    const dnTransliteration = h(
+      'div.devnagri-transliteration.translittext',
+      row.Transliteration.Devnagri || '',
+    );
+    const enTransliteration = h(
+      'div.english-transliteration.translittext',
+      row.Transliteration.English || '',
+    );
+    const transliterationsContainer = h(
```

Comment:
this approach doesn't scale well IMO. I'm fine with it for now since it will work, but I'd like to revisit in the future.

Reply:
What approach would you suggest?

Reply:
react 😄

Reply:
No need to change now.